### ISSUE TITLE
Working with multiple losses and their weights in keras

### ISSUE THREAD

**Issue by zeeshannisar** [keras-team/keras#11985]
Training a GAN model using `train_on_batch` with multiple losses, can I use random `loss_weights` while compiling a model or is there some specific strategy to use these loss weights as mentioned [Here](https://stackoverflow.com/questions/48620057/keras-loss-weights?answertab=oldest#tab-top). In my problem, `mean_sqaured_error` is a loss function for generated_image and original_image and `binary_crossentropy ` is a classification loss function for 0 and 1 class.

`model.compile(optimizer=optimizer, loss=['mean_squared_error', 'binary_crossentropy'], loss_weights=[100,1])`

**Comment by Tiaspetto** [comment]
Not quite sure what's your question, did u mean that u want to do something like loss_weights =[random1, random2]?

**Comment by msymp** [comment]
Hello @zeeshannisar , can you please provide us your system details in the below template.

System information:

- Have I written custom code (as opposed to using example directory):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow backend (yes / no):
- TensorFlow version (CPU / GPU):
- Python version:
- CUDA/cuDNN version:
- GPU model and memory (Full Specifications):

You can obtain the TensorFlow version with:
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

Describe the problem:
Describe the problem clearly here. Be sure to convey here why it's a bug in Keras or a feature request.

Source code / logs:
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by ParikhKadam** [comment]
Yes.. you can use any values as loss weights but keep in mind that the values you select must help you increase your models accuracy. This isn't an issue in Keras. The question suits the most on stack. You should close this issue now..

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will open a new issue. Thanks!
